install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-focal

```diff
@@ -0,0 +1,83 @@
+securedrop-app-code (1.5.0~rc1+focal) focal; urgency=medium
+
+  * 
+
+ -- SecureDrop Team <securedrop@freedom.press>  Thu, 18 Jun 2020 21:58:23 +0000
+
+securedrop-app-code (1.4.0+xenial) xenial; urgency=medium
+
+  * See changelog.md 
+
+ -- SecureDrop Team <securedrop@freedom.press>  Wed, 17 Jun 2020 21:35:57 +0000
+
+securedrop-app-code (1.3.0+xenial) xenial; urgency=medium
+
+  * See changelog.md
+
+ -- SecureDrop Team <securedrop@freedom.press>  Tue, 12 May 2020 18:37:42 +0000
+
+securedrop-app-code (1.2.2+xenial) xenial; urgency=medium
```

Comment:
Focal changelog should not include historical xenial versions. Remove those, and start with the current/latest version for the focal history. (We did the same when moving from trusty to xenial.)